src/models/auth/Client.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+
+const clientSchema = mongoose.Schema(
+  {
+    id: String, // client_id
```

Comment:
Again, I put this in - I think it was mostly for clarity in terms of using the terms client id and client secret all the time this was just making it clear, but yeah not really necessary